### ISSUE TITLE
How to recover priv key/location on Metamask FF

### ISSUE THREAD

**Issue by githubimpulse** [MetaMask/metamask-extension#1947]
Hi guys,
Unable to get MM on FF to prompt for my existing account. Didnt install a new one, but when PC crashed, something mayve happened. This less than 1 week from Install and hadnt got time to save seed/  words , backup etc.
I do have the private key locally  - just dont know where its at for FF. Scoured all over but no clue for an equivalent to the chrome path as given in issue #1372 and also  #1661 - though that answer drifted away and  wasnt accessible to the main github community. 

So in a nutshell, I have the  password  for my vault/DEN/account and priv key created by MM somewhere in my PC. Just that MM seems to want to create a new DEN and I fear overwriting the old If i just create a new DEN.

I'd be happy to write a quick guide for future FF users who stumble down that path  once solved.
Thx in advance
![m_barefox](https://user-images.githubusercontent.com/31196469/29503344-e7e50500-8604-11e7-83b0-971a0e94a978.png)

**Comment by koraysels** [comment]
i have the same problem! did you manage to fix it ?
